status-im/status-react
# Follow up replacing old skeleton loader with the latest version

fixes #17103

Follow up from #16865 

### Pull Request Summary
This pull request focuses on enhancing the user experience by replacing our legacy skeleton loaders with the latest version. Additionally, it introduces a refined approach for deprecating outdated components and functions to maintain a cleaner codebase.

### Key Changes
Replace existing skeleton loaders with the latest skeleton-list component for improved performance and usability.
Implement a standardized deprecation method, using either naming conventions or metadata annotations, for phasing out old components and functions.

### Review notes
###### We are currently in the process of refining our methodology for deprecation, as outlined in this evolving document. Your review and contributions would be highly beneficial for its improvement. Please, check fixes #17177 for further details

### Testing notes
All tests are expected to pass upon completion of this feature.

#### Platforms

- Android
- iOS

#### Areas that may be impacted

Please be advised that the changes in this update could significantly impact the chat and community features.

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test
To observe the functionality, navigate to either a community chat or a one-on-one conversation. You should encounter a skeleton loader prior to the full rendering of the conversation.


##### Light mode

https://github.com/status-im/status-mobile/assets/5999878/abdbe0b7-c317-4595-afbc-8d1bc5f34fe2


##### Dark mode
[to be updated]
